systest/artifacts/readme.md

```diff
@@ -0,0 +1,23 @@
+helm repo add grafana https://grafana.github.io/helm-charts
+helm repo update
+
+helm install grafana grafana/grafana --values ./grafana.yaml
+helm install loki grafana/loki --values ./loki.yaml
+helm install promtail grafana/promtail --values ./promtail.yaml
```

Comment:
i was using chart `grafana/loki-stack ` 

```
helm upgrade --install loki grafana/loki-stack  --set grafana.enabled=true,prometheus.enabled=true,prometheus.alertmanager.persistentVolume.enabled=false,prometheus.server.persistentVolume.enabled=false,loki.persistence.enabled=true,loki.persistence.storageClassName=standard,loki.persistence.size=20Gi -f promtail.yaml
```

it automatically adds loki source to the grafana

what about cri-o pipeline, did you enable it or logs collector works without it?

Reply:
The collector seems to work without the cri-o pipeline.